example/lib/presentation/samples/line/line_chart_sample3.dart

```diff
@@ -206,7 +206,22 @@ class _LineChartSample3State extends State<LineChartSample3> {
                     }).toList();
                   },
                   touchTooltipData: LineTouchTooltipData(
-                    tooltipBgColor: widget.tooltipBgColor,
+                    getTooltipColor: (touchedSpot) {
+                      Color bgColor = switch ((
+                        touchedSpot.x.toInt(),
+                        touchedSpot.y.toDouble()
+                      )) {
+                        (0, 1.3) => AppColors.contentColorPurple.withAlpha(100),
```

Comment:
Also here, please use the previous color

Reply:
Done